ext/intl/breakiterator/rulebasedbreakiterator_methods.h

```diff
@@ -17,14 +17,4 @@
 
 #include <php.h>
 
-PHP_METHOD(IntlRuleBasedBreakIterator, __construct);
```

Comment:
should the whole file be deleted, right?

Reply:
Right.